core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -594,6 +601,8 @@ public interface MetricsProducer {
 
   Logger LOG = LoggerFactory.getLogger(MetricsProducer.class);
 
+  String METRICS_APP_PREFIX = "accumulo.app.";
```

Comment:
"app" sounds like "mobile app", and sounds a bit "trendy" and less serious/scientific. Maybe "server"? Or just "accumulo."? Or "accumulo.application"?

Reply:
addressed in 674a9da956